all-repos.yaml

```diff
@@ -214,3 +214,4 @@
 - https://github.com/shssoichiro/oxipng
 - https://github.com/datarootsio/databooks
 - https://github.com/standard/standard
+- https://github.com/Sceptre/sceptrelint
```

Comment:
https://github.com/Sceptre/sceptrelint/blob/aee222f5565cb91748b8af23f09a0916600888e1/.pre-commit-hooks.yaml#L5-L6

- `pass_filenames: true` is the default
- `always_run: true` generally should not be set -- pre-commit will pass filenames for changed things to the tool so it runs selectively (otherwise when no files are changed the tool will receive 0 filenames and _probably_ do the wrong thing)

Reply:
thanks for the tip @asottile .  Those configs have been removed.